library/ubuntu

```diff
@@ -87,22 +87,25 @@ ppc64le-GitCommit: 54f9f186c9af571f1d874614b0a17512166b60aa
 s390x-GitFetch: refs/tags/dist-noble-s390x-20240605-17a304a2
 s390x-GitCommit: 17a304a2f858f7401179b7bd7bb15340e8ee2d51
 
-# 20240527 (oracular)
-Tags: 24.10, oracular-20240527, oracular, devel
-Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
+# 20240617 (oracular)
+Tags: 24.10, oracular-20240617, oracular, devel
+Architectures: amd64, arm32v7, arm64v8, ppc64le, riscv64, s390x
```

Comment:
note the re-introduction of riscv64

Reply:
@cjdcordeiro this is great to see! do we expect to see riscv64 images for Ubuntu Noble anytime soon?

Reply:
:)

we're working on it @gdams

Reply:
Great I'll keep an eye out!